src/classes/project_data.py

```diff
@@ -257,20 +253,17 @@ def new(self):
         import openshot
 
         # Try to load user default project
-        if os.path.exists(self.user_project_filepath):
+        if os.path.exists(info.USER_DEFAULT_PROJECT):
             try:
-                self._data = self.read_from_file(self.user_project_filepath)
+                self._data = self.read_from_file(info.USER_DEFAULT_PROJECT)
             except (FileNotFoundError, PermissionError) as ex:
-                log.warning("Unable to load defaults from {}: {}".format(
-                    new_project_file, ex))
+                log.warning("Unable to load user project defaults from {}: {}".format(info.USER_DEFAULT_PROJECT, ex))
             except Exception:
                 raise
             else:
-                log.info("Loaded user project defaults from {}".format(
-                    self.user_project_filepath))
-
-        # Fall back to OpenShot defaults, if user defaults didn't load
-        if not self._data:
```

Comment:
if not self._data would always be true for an existing project loaded... and thus, it would never clear your existing project when clicking "New Project"

Reply:
I really cocked this one up, huh? (And you didn't even mention that the exception log message tried to output `new_project_file`, despite no such variable existing.)

I think the _idea_ with the `if not self._data:` was that I'd test to see whether `self.read_from_file()` **successfully** loaded the user's default project data, so that if it failed due to bad syntax or whatever I'd just roll with it. Which is a nice enough idea, I suppose, if only I'd remembered to set `self._data = None` at the top of the method!